yatsm/cli/pixel.py

```diff
@@ -190,15 +191,27 @@ def pixel(ctx, config, px, py, band, plot, ylim, style, cmap,
         )
 
 
-def plot_TS(dates, y):
+def plot_TS(dates, y, seasons):
     """ Create a standard timeseries plot
 
     Args:
         dates (iterable): sequence of datetime
         y (np.ndarray): variable to plot
     """
     # Plot data
-    plt.scatter(dates, y, c='k', marker='o', edgecolors='none', s=35)
+    if seasons == []:
+        plt.scatter(dates, y, c='k', marker='o', edgecolors='none', s=35)
+    else:
+        for date in dates:
```

Comment:
I didn't notice originally but you're looping over every single observation and triggering a plot command, which could take a very long time. Better to define all of the members of a season in one go (use `np.in1d`) and only call `plt.plot` once per season.